huaweicloud/config/endpoints.go

```diff
@@ -313,6 +314,11 @@ var allServiceCatalog = map[string]ServiceCatalog{
 		Version: "mysql/v3",
 		Product: "GaussDB",
 	},
+	"gaussdb-tag": {
```

Comment:
it will replace gaussdb in the future， s/gaussdb-tag/gaussdbV3

Reply:
@hmbig2  we will use gaussdb directly when replacing it, so it's fine here.